store/tikv/lock_resolver.go

```diff
@@ -150,6 +153,7 @@ type Lock struct {
 	TxnSize         uint64
 	LockType        kvrpcpb.Op
 	LockForUpdateTS uint64
+	MinCommitTS     uint64
 }
```

Comment:
We need to extract `UseAsyncCommit` in `LockInfo` to `Lock`

Reply:
Where do we need to use it?

Reply:
nvm, saw comment at top